Formula/mpdas.rb

```diff
@@ -21,6 +21,35 @@ def install
     etc.install "mpdasrc.example"
   end
 
+  def caveats; <<~EOS
+    The example configuration file was placed in #{etc}/mpdasrc.example
+  EOS
+  end
```

Comment:
Why did you add this caveat? Caveats should only be for homebrew specific issues/quirks. At the very least the text should be in the present perfect tense ("is placed") because the caveat is printed during `brew info` as well as after installation.

Reply:
Because I saw mpdscribble have a caveat that print the path of configuration file, and during configure mpdas I tried to find the configuration file by listing files in the package and noticed mpdasrc.example, thought it could be useful for ppl just installed this package and don't know how to configure it.
But you are right, printing this is not necessary and could be annoying, the example configuration file does not have any comments, ppl can create the configuration file by looking at manpage. I will remove the caveat.

Reply:
sorry I'm not familiar with git pull request, the commit message of the second commit does not looks fit.. Can you squash these commit while merging?